docs/src/content/docs/commands/JSON.TYPE.md

```diff
@@ -66,44 +75,44 @@ If the path does not exist within the JSON document, the command will return `nu
 ### Example 1: Basic Usage
 
 ```shell
-127.0.0.1:6379> JSON.SET user:1001 $ '{"name": "John", "age": 30, "address": {"city": "New York", "zip": "10001"}}'
+127.0.0.1:7379> JSON.SET user:1001 $ '{"name": "John", "age": 30, "address": {"city": "New York", "zip": "10001"}}'
 OK
-127.0.0.1:6379> JSON.TYPE user:1001 $.name
+127.0.0.1:7379> JSON.TYPE user:1001 $.name
 "string"
-127.0.0.1:6379> JSON.TYPE user:1001 $.age
+127.0.0.1:7379> JSON.TYPE user:1001 $.age
 "number"
-127.0.0.1:6379> JSON.TYPE user:1001 $.address
+127.0.0.1:7379> JSON.TYPE user:1001 $.address
 "object"
-127.0.0.1:6379> JSON.TYPE user:1001 $.address.city
+127.0.0.1:7379> JSON.TYPE user:1001 $.address.city
 "string"
 ```
 
 ### Example 2: Non-Existent Path
 
 ```shell
-127.0.0.1:6379> JSON.TYPE user:1001 $.nonexistent
+127.0.0.1:7379> JSON.TYPE user:1001 $.nonexistent
 (null)
 ```
 
 ### Example 3: Key Does Not Exist
 
 ```shell
-127.0.0.1:6379> JSON.TYPE user:9999 $.name
+127.0.0.1:7379> JSON.TYPE user:9999 $.name
 (nil)
 ```
 
 ### Example 4: Invalid JSONPath
 
 ```shell
-127.0.0.1:6379> JSON.TYPE user:1001 $..name
+127.0.0.1:7379> JSON.TYPE user:1001 $..name
 (error) ERR wrong number of arguments for 'json.type' command
```

Comment:
I think there is some discrepancy in the heading for this example and the error message shown here. Please cross check once in the latest version of DiceDB.
Thanks